interfaces/cython/cantera/examples/onedim/premixed_counterflow_twin_flame.py

```diff
@@ -23,42 +62,63 @@ def solveOpposedFlame(oppFlame, massFlux=0.12, loglevel=1,
+    
+    return np.max(oppFlame.T), K, strainRatePoint
 
-
-# Use the standard GRI3.0 Mechanism for CH4
+# Select the reaction mechanism                                                                        
 gas = ct.Solution('gri30.cti')
 
-# Create a CH4/Air premixed mixture with equivalence ratio=0.75, and at room
-# temperature and pressure.
+# Create a CH4/Air premixed mixture with equivalence ratio=0.75, and at room temperature and pressure
+
 gas.set_equivalence_ratio(0.75, 'CH4', {'O2':1.0, 'N2':3.76})
 gas.TP = 300, ct.one_atm
 
-# Set the velocity
-axial_velocity = 0.25 # in m/s
+# Set the velocity                                                                                                     
+axial_velocity = 2.0 # in m/s                                                                                          
 
-# Domain half-width of 2.5 cm, meaning the whole domain is 5 cm wide
+# Domain half-width of 2.5 cm, meaning the whole domain is 5 cm wide                                                   
 width = 0.025
 
-# Done with initial conditions
-
-# Compute the mass flux, as this is what the Flame object requires
+# Done with initial conditions                                                                                         
+# Compute the mass flux, as this is what the Flame object requires                                                     
 massFlux = gas.density * axial_velocity # units kg/m2/s
-# Create the flame object
+
+# Create the flame object                                                                                              
 oppFlame = ct.CounterflowTwinPremixedFlame(gas, width=width)
 
-# Now run the solver
+# Now run the solver  
 
-# The solver returns the peak temperature and strain rate. You can plot/see all
-# state space variables by calling oppFlame.foo where foo is T, Y[i] or whatever
-# The spatial variable (distance in meters) is in oppFlame.grid Thus to plot
+# The solver returns the peak temperature, strain rate and the point which we 
+# ascribe to the characteristic strain rate.
+
+# You can plot/see all                                         
+# state space variables by calling oppFlame.foo where foo is T, Y[i] or whatever                                       
+# The spatial variable (distance in meters) is in oppFlame.grid Thus to plot                                          
 # temperature vs distance, use oppFlame.grid and oppFlame.T
-(T, K) = solveOpposedFlame(oppFlame, massFlux)
 
-print("Peak temperature: {0}".format(T))
-print("Strain Rate: {0}".format(K))
+(T, K, strainRatePoint) = solveOpposedFlame(oppFlame, massFlux, loglevel=1)
+
+Sc = computeConsumptionSpeed(oppFlame)
+
+print("Peak temperature: {0} K".format(T))
+print("Strain Rate: {0} 1/s".format(K))
+print("Consumption Speed: {0} cm/s".format(Sc*100))
 oppFlame.write_csv("premixed_twin_flame.csv", quiet=False)
+
+"""
+#Uncomment to generate plots and verify the solution
```

Comment:
Jupyter notebooks certainly have their uses, but these examples are distributed with Cantera as Python scripts, so that a user could run, say, `python /path/to/site-packages/cantera/examples/onedim/burner_flame.py` to check that their install is working. In this case, the `--plot` option is very useful. I do not think this usage is archaic at all...